Build/libHttpClient.141.Android.Java/app/src/main/java/com/xbox/httpClient/HttpClientRequest.java

```diff
@@ -20,6 +23,12 @@ public HttpClientRequest() {
         requestBuilder = new Request.Builder();
     }
 
+    public static boolean isNetworkAvailable(Context context) {
+        ConnectivityManager connectivityManager = (ConnectivityManager) context.getSystemService(Context.CONNECTIVITY_SERVICE);
+        NetworkInfo activeNetworkInfo = connectivityManager.getActiveNetworkInfo();
+        return activeNetworkInfo != null && activeNetworkInfo.isConnected();
```

Comment:
Are there any concerns of platform support here on older API levels?

Reply:
Nope this API has existed since level 1